examples/tutorials/core_api_pytorch_mnist/model_def_distributed.py

```diff
@@ -18,6 +18,8 @@
 from torchvision import datasets, transforms
 
 import determined as det
+import os
+import filelock
```

Comment:
nitpick: python isort will complain these need to be next to other standard library imports in lines 6 & 7.

Reply:
fixing